### DISCUSSION TITLE
Support for Zigtiger dimmer

### DISCUSSION THREAD

**Discussion by adi-dev** [Koenkk/zigbee2mqtt#9301] [Devices]
Just bought a Zigtiger dimmer on [eBay](https://www.ebay.co.uk/itm/373612630279?_trkparms=amclksrc%3DITM%26aid%3D111001%26algo%3DREC.SEED%26ao%3D1%26asc%3D20160908105057%26meid%3D0abeb13a4b1045edb2f92139b49259ce%26pid%3D100675%26rk%3D1%26rkt%3D15%26sd%3D373612630279%26itm%3D373612630279%26pmt%3D1%26noa%3D1%26pg%3D2380057&_trksid=p2380057.c100675.m4236&_trkparms=pageci%3A41ac8fb0-34ac-11ec-aa67-2e8ed4635995%7Cparentrq%3Ab19e0ed917c0acf38221ec7ffff8b5d6%7Ciid%3A1). It won't connect, it doesn't even show anything in the log, as [this page](https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html) suggests. Is there anything, I could try to get this working. I like this dimmer as this is my first dimmer doesn't flicker the lights. I have few other devices on Zigbee network, no issues.

**Comment by BeoQ** [comment]
The dimmer from the ZigTiger brand is not a Zigbee device at all.
It has no wireless electronics at all.
